week0/homework/Person.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function(name,age) {
+    let arr = [];
+    this.name= name;
+    this.age = age;
+    this.addPhone = function(phone) {
+         arr.push({"Name":this.name,"Age":this.age,"Phone":phone});
+    }
+    
+    this.call = function() {
+        var phone = arr.map(function(o){return o.Phone});
+        if (phone != "") {
+            return "Calling " + this.name + " at " + arr[0]["Phone"] + " ..."; 
+        } else { 
+            return this.name + " has no phone number saved.";
+        }
+    }
+    this.birthday = function() {
+        return "Wishing " + this.name + " a happy 29th birthday!";
```

Comment:
It wasn't clear in the homework, but you should've incremented the age here (Jimmy was 28 in the example), and not hard-coded the display for birthday. For example see: https://github.com/HackYourFuture-CPH/Node.js/pull/3/commits/e69bffdf5a976195ab6cb0cf7e0c9ce67ef37a72 (I gave the same feedback there previously).